test/services/katello/pulp3/repository/python/python_test.rb

```diff
@@ -52,6 +52,13 @@ def test_index_on_sync
             post_unit_count = Katello::GenericContentUnit.all.count
             post_unit_repository_count = Katello::RepositoryGenericContentUnit.where(:repository_id => @repo.id).count
 
+            unit = Katello::GenericContentUnit.all.first
```

Comment:
doing a .first like this is very dangerous, it might work for now, but in the future could break in unexpected/transient ways.  It'd be better to look it up by the repository:  @repo.generic_content_units.first

Reply:
because if there were more generic content units added in the future, Katello::GenericContentUnit.all would return something else?

Reply:
@jlsherrill small question ^^

Reply:
i was more thinking that if we added fixtures later on that cause entries to appear in the database, it could fail